rucio-dataset-mon/src/go/rucio-go-datatable/configs/setup.go

```diff
@@ -0,0 +1,40 @@
+package configs
+
+import (
+	"context"
+	"fmt"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"log"
+	"time"
+)
+
+func ConnectDB() *mongo.Client {
```

Comment:
A good practice to supply comments for all public functions (functions started with capital letter). This will provide auto-documentation for the code. I also encourage you to run your code against https://goreportcard.com/ which will give you hints how to improve it. Anyway, for comments it should looks like:
```
// ConnectDB provides bla-bla-bla
func ConnecDB() *mongo.Client {
```
The comment should start with name of the function.